docs/book/src/clusterctl/overview.md

```diff
@@ -20,11 +20,15 @@ Below you can find a list of main clusterctl commands:
 * [`clusterctl move`](commands/move.md) Move Cluster API objects and all their dependencies between management clusters.
 * [`clusterctl alpha rollout`](commands/alpha-rollout.md) Manages the rollout of Cluster API resources. For example: MachineDeployments.
 
+### GitHub Personal Access Token
```

Comment:
I will add some context, a short blurb about why/when clusterctl is accessing GitHub, and explain that for normal usage the free tier is enough, but for intensive usage a token is required.
Also, worth to link this paragraph from configuration and from the clusterctl init page as well